### ISSUE TITLE
Reproduction of results

### ISSUE THREAD

**Issue by Huertas97** [UKPLab/sentence-transformers#519]
Hi @nreimers!

I have been trying to reproduce your results from your experiments off Multilungual Semantic Textual Similarity [https://arxiv.org/abs/2004.09813]( https://arxiv.org/abs/2004.09813 ) (Table 1 and Table 2). 

![imagen](https://user-images.githubusercontent.com/56938752/97070221-f73bad00-15d6-11eb-9901-ec6d79ed5e4a.png)

![imagen](https://user-images.githubusercontent.com/56938752/97070271-6e714100-15d7-11eb-8fc0-bc82a7c28741.png)

Although, I've obtained quite similar results, there are small differences between my values on STS Benchmark 2017 extended: 

![imagen](https://user-images.githubusercontent.com/56938752/97071246-7339f300-15df-11eb-950e-5c83e03bd200.png)

I'm using the pre-trained multilingual models evaluating them using parts of the code from make_multilingual.py. I'm using a `inference_batch_size = 16 ` in the evaluator. Is this normal? Where can be my mistake? 

Thanks in advance!

**Comment by nreimers** [comment]
Hi,
in the paper, I used a model that was just trained on the 50+ languages mentioned. However, I think the released xlm-r-distilroberta-base-paraphrase-v1 was trained on more languages: I added some multi-lingual vocab for more languages, so that it works better for languages without parallel sentence data.

Note: STS2017 test set is with 250 sentences rather small. A few changes like pair on position 83 is now on position 87 can then have already a "big" impact on the computed scores.
